src/rest/remote-api.ts

```diff
@@ -192,7 +192,11 @@ export interface IRemoteAPI {
     /**
      * Returns list of kubernetes namespace.
      */
-    getKubernetesNamespace<T = KubernetesNamespace[]>(): Promise<T>;
+    getKubernetesNamespaces<T = KubernetesNamespace[]>(): Promise<T>;
+    /**
+     * Returns kubernetes namespace.
```

Comment:
Please add more info, possibly copy paste from che server swagger, since it not only returns namespace

Reply:
Thanks. I have fixed it.